From 71e2c55197ffa63a40b92a966aaff53bc2e04e11 Mon Sep 17 00:00:00 2001
From: Prashanth <prashanth.kamanoor@external.marelli.com>
Date: Sun, 8 May 2022 15:48:12 +0530
Subject: [PATCH 2/3] commit 2 on feature

---
 file/document | 1 +
 1 file changed, 1 insertion(+)

diff --git a/file/document b/file/document
index 862b157..537a520 100644
--- a/file/document
+++ b/file/document
@@ -1,3 +1,4 @@
 my name is prashanth
 im a softwate engineer
 i love pandu
+i love bhagya
-- 
2.34.1.windows.1

