From 4621355ce277c759c0237a568f742c416a706966 Mon Sep 17 00:00:00 2001
From: Prashanth <prashanth.kamanoor@external.marelli.com>
Date: Sun, 8 May 2022 15:19:20 +0530
Subject: [PATCH 1/3] commit 1 on feature

---
 file/document | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/file/document b/file/document
index 7fc37e9..862b157 100644
--- a/file/document
+++ b/file/document
@@ -1,3 +1,3 @@
 my name is prashanth
 im a softwate engineer
-i love sitara
+i love pandu
-- 
2.34.1.windows.1


From 71e2c55197ffa63a40b92a966aaff53bc2e04e11 Mon Sep 17 00:00:00 2001
From: Prashanth <prashanth.kamanoor@external.marelli.com>
Date: Sun, 8 May 2022 15:48:12 +0530
Subject: [PATCH 2/3] commit 2 on feature

---
 file/document | 1 +
 1 file changed, 1 insertion(+)

diff --git a/file/document b/file/document
index 862b157..537a520 100644
--- a/file/document
+++ b/file/document
@@ -1,3 +1,4 @@
 my name is prashanth
 im a softwate engineer
 i love pandu
+i love bhagya
-- 
2.34.1.windows.1


From 04ebb9c8e4b1680592e2e1caf29258fb30e0fb2a Mon Sep 17 00:00:00 2001
From: Prashanth <prashanth.kamanoor@external.marelli.com>
Date: Sun, 8 May 2022 15:50:53 +0530
Subject: [PATCH 3/3] commit 3 on feature

---
 file/document | 1 +
 1 file changed, 1 insertion(+)

diff --git a/file/document b/file/document
index 537a520..e83aab0 100644
--- a/file/document
+++ b/file/document
@@ -2,3 +2,4 @@ my name is prashanth
 im a softwate engineer
 i love pandu
 i love bhagya
+i love preethi
-- 
2.34.1.windows.1

